### DISCUSSION TITLE
Ingot 106 and up yall should work on Ingot 106 and up

### DISCUSSION THREAD

**Discussion by GlizxyBoi** [3kh0/ext-remover#604] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
yes do it

**Comment by obkduishinijf** [comment]
explain how that would work

**Comment by Aka-but-nice** [comment, marked as answer]
> yes do it

Dude, not possible, trust me on this one, if we could just " work on Ingot 106 and up" it would've already been done

**Comment by funkymokey** [comment]
You cant just make it work, i have 0% coding skill but I'm not this idiotic

**Comment by TheGooferFox** [comment]
Yk what ya its sooooo easy to just fucking go from 106 and up u do realize its restricted because of the API and guess who fucking controls it dumbass fr

yes I want toxic on purpose I am honestly bored lol
